app/lib/routes/api.rb

```diff
@@ -0,0 +1,87 @@
+require 'json'
+Pakyow::App.routes(:api) do
+  include SharedRoutes
+
+  expand :restful, :api, '/api' do
+    collection do
+
+      expand :restful, :v1, '/v1' do
+        collection do
+
+          expand :restful, :groups, '/groups' do
+            member do
+
+              expand :restful, :events, '/events' do
+                get 'schedule' do
+                  # "approved":true,
+                  # "cwn":88,
+                  # "timestamp":"2017-01-09T20:27:22.161Z",
+                  # "group":"Designer's Corner",
+                  # "title":"CoWorking Night Tshirt Competition",
+                  # "description":"Come in to get some last minute ideas for your entry to the CoWorking Night Tshirt competition! ",
+                  # "date":"2017-01-11T06:00:00.000Z",
+                  # "time_req_form":"8:00 PM",
+                  # "time_req":"1899-12-31T01:48:52.000Z",
+                  # "room_req":"Milky Way Row",
+                  # "start_time":"2017-01-12T02:00:00.000Z",
+                  # "end_time":"2017-01-12T03:00:00.000Z"
+
+                  #For now, we'll keep this only exposed for cwn
+                  cwn = Group.where("id = ?", params[:groups_id]).first
```

Comment:
You've got a point here. Initially I implemented this to be extensible to other groups. Then, somewhere along the way, I decided it would only be exposed for CWN. I'll make the necessary changes that reflect that diversion